### DISCUSSION TITLE
How can I change the art player subtitle color and font?

### DISCUSSION THREAD

**Discussion by kanekabkz** [AlistGo/alist#926] [Q&A]
I can't change the artplayer subtitle font, color, and subtitle size. How can I change it? I think it would be great if you put a setting for this in the manage section. Subtitles look much nicer in different indexes using artplayer. Btw, i loved Alist's performance and stability. also italics is not supported by artplayer i think. Is there any way I can get past this?
![ss](https://user-images.githubusercontent.com/79414108/162611799-0c571fff-e0cd-4d1c-ba5a-c3e8bd75a3b2.png)

**Comment by xhofe** [comment]
You can add a custom css code to the `customize head` field to change the style of the subtitle, for example:
```html
<style>
.art-video-player .art-subtitle {
    color: #FEB2B2 !important ;
    font-size: 40px !important;
    font-style: italic !important;
}
</style>
```

**Reply by kanekabkz** [reply]
Really thank you very much, but I will not make the font italic. There are italic texts in some parts of the srt or vtt files, and they appear as code, not italic. and can i add shadow to the subtitle?

**Reply by kanekabkz** [reply]
and can i add shadow to the subtitle?

**Reply by xhofe** [reply]
You can try. In theory, you can add any style to subtitles via css code.
